Summit/2022/program.html

```diff
@@ -55,6 +55,7 @@
     title = "Welcome"
     presenter = "June Andronick, Darren Cofer"
     affiliation = "Proofcraft, Collins Aerospace"
+    url = "https://proofcraft.systems/, https://www.collinsaerospace.com/"
```

Comment:
Should we call it `urls` if there can be multiple?

How are these processed, btw? `,` is a legal character in an URL, i.e. `https://proofcraft.systems/,` is an URL. I don't think we'd have any of those here, but we might want just a space separated list to be sure we can express all URLs.

Reply:
> Should we call it `urls` if there can be multiple?

Can do. I'll change it to urls.

> How are these processed, btw? `,` is a legal character in an URL, i.e. `https://proofcraft.systems/,` is an URL. I don't think we'd have any of those here, but we might want just a space separated list to be sure we can express all URLs.

They are parsed as comma-separated strings. I didn't realise that commas were legal characters in urls, and there aren't any in the summit-word-cloud list. I will parse them by using a whitespace as a separator, just in case.